crates/cast/src/helpers/response_structs.rs

```diff
@@ -19,9 +19,9 @@ pub struct DeclareResponse {
     pub transaction_hash: FieldElement,
 }
 
-#[derive(Serialize)]
+#[derive(Serialize, Clone, Debug)]
 pub struct CallResponse {
-    pub response: String,
+    pub data: Vec<FieldElement>,
```

Comment:
```suggestion
    pub response: Vec<FieldElement>,
```

Let's maybe keep the previous name? I guess it is more self-explanatory

Reply:
changed